test/browser/msw-api/setup-worker/start/error.mocks.ts

```diff
@@ -7,7 +7,8 @@ const worker = setupWorker(
   }),
 )
 
-// @ts-ignore
-window.msw = {
-  worker,
-}
+Object.assign(window, {
```

Comment:
instead of this, we could (and maybe should) also extend the window types in our tests to account for the few additional properties, and `invariant` them to ensure they're defined

Reply:
We do that for test files themselves but not for these examples. We can use the same `declare` statements in those too but it's a bit repetitive.